src/bakery/Bakery.cpp

```diff
@@ -0,0 +1,77 @@
+//
+// Created by Domingos Santos on 05/05/2021.
+//
+
+#include <fstream>
+#include <iostream>
+#include "Bakery.h"
+#include "../utils/GraphBuilder.h"
+
+using namespace std;
+
+
+
+Bakery::Bakery(const vector<Client> &clients, const vector<Van> &vans, Vertex *startingVertex, double radius,
+               int maxDelay, int maxTimeBefore) : clients(clients), vans(vans), startingVertex(startingVertex),
+                                                  radius(radius), maxDelay(maxDelay), maxTimeBefore(maxTimeBefore) {}
+
+Bakery::Bakery(string filePath) {
+    ifstream fin;
+
+    fin.open(filePath);
+
+    if(!fin.is_open())
+        throw runtime_error("File not found (Bakery)");
+
+    string graphPathName, clientName;
+
+    int latitude, longitude, toleranceDelay, toleranceBefore, numVans, vanCapacity, deliveryTime;
+    double radius;
+    char token;
+
+    fin >> graphPathName >> token >> latitude >> token >> longitude >> token >> radius >> toleranceDelay >> toleranceBefore >> numVans;
```

Comment:
We're reading the radius, toleranceDelay and toleranceBefore to local variables instead of the class ones. Just delete the local ones and change them by the class atributes, they don't serve any purpose